js/util/browser/dom.js

```diff
@@ -51,10 +51,22 @@ exports.suppressClick = function() {
     }, 0);
 };
 
-exports.mousePos = function (el, e) {
+var _cachedOffset;
```

Comment:
No need for `_` here — the variable stays private in a module, and we don't use this convention for variables anywhere else in the code (it's only for private methods/properties of classes).